tests/attestation_plugin/plugin/tests.c

```diff
@@ -603,21 +618,16 @@ static void _test_time(
     oe_datetime_t* from,
     oe_datetime_t* until)
 {
+    oe_result_t result;
     oe_datetime_t tmp;
 
-    OE_TEST_CODE(
-        oe_verify_sgx_quote(
-            report_body, report_body_size, collaterals, collaterals_size, from),
-        OE_OK);
+    result = oe_verify_sgx_quote(
+        report_body, report_body_size, collaterals, collaterals_size, from);
+    OE_TEST(result == OE_OK || result == OE_TCB_LEVEL_INVALID);
```

Comment:
When would the result be OE_TCB_LEVEL_INVALID?

Reply:
`oe_verify_sgx_quote` calls `oe_verify_quote_with_sgx_endorsements` to verify the quote. When tcb level configuration needed or is out-of-date, the result will be `OE_TCB_LEVEL_INVALID`. The result here should be consistent with [Line.895](https://github.com/openenclave/openenclave/blob/a399e38f53e05d140e9e4f672482c0dcec784bdd/tests/attestation_plugin/plugin/tests.c#L895) as they are verify the same SGX quote.